packages/server/worker/src/lib/consume/job-queue-worker.ts

```diff
@@ -47,21 +48,21 @@ export const jobQueueWorker = (log: FastifyBaseLogger) => ({
                 assertNotNullOrUndefined(jobId, 'jobId')
                 const { shouldRateLimit } = await workerJobRateLimiter(log).shouldBeLimited(jobId, job.data)
                 if (shouldRateLimit) {
-                    const baseDelay = Math.min(600, 20 * Math.pow(2, job.attemptsStarted))
-                    const randomFactor = 0.6 + Math.random() * 0.4
-                    const delayInSeconds = Math.round(baseDelay * randomFactor)
-                    await job.moveToDelayed(
-                        dayjs().add(delayInSeconds, 'seconds').valueOf(),
-                        token,
-                    )
+                    const redis = await workerRedisConnections.useExisting()
+                    const key = projectWaitingSetKey(job.data.projectId!)
+
+                    const member = JSON.stringify({ id: job.id!, data: job.data })
```

Comment:
We already adding the job to the throttledQueue, there is no need to store all the data, we should just store the `jobId` and when we pop it out we can fetch the data from the queue and insert it in the main queue.